deployments/installer/install.ps1

```diff
@@ -0,0 +1,305 @@
+# The following comment block acts as usage for powershell scripts
+# you can view it by passing the script as an argument to the cmdlet 'Get-Help'
+# To view the paremeter documentation invoke Get-Help with the option '-Detailed'
+# ex. PS C:\> Get-Help "<path to script>\install.ps1" -Detailed
+
+<#
+.SYNOPSIS
+    Installs the SignalFx Agent from the package repos.
+.DESCRIPTION
+    Installs the SignalFx Agent from the package repos.  If access_token is not
+    provided, it will prompted for on stdin.  If you want to view full documentation

```

Comment:
I would just say "it will be prompted for on the console" since Windows people might not be as familiar with stdin.

Regardless, there needs to be something between "will" and "prompted".